File: Assignment__26/Assignment2.c
#include<stdio.h>


int  CountChar(char *str, char ch)
{
    int iCount = 0;
    while(*str != '\0')
    {
        if(*str == ch)
        {
           iCount++; 
        }
        str++;
    }
    return iCount;
}

int main()
{   
    char Arr[50];
    char cValue = '\0';
    int iRet = 0;

    printf("Enter String :");
    scanf("%[^'\n']",Arr);

    printf("Enter Charecter :");
    scanf(" %c", &cValue);

    iRet = CountChar(Arr,cValue);

    printf("Frequency of that charecter : %d\n",iRet);


    return 0;
}

File: Assignment____44/Assignment2.java
//  Input : 9

//  3   2   5   9
//  4   3   2   2
//  8   4   1   9
//  3   9   7   5

//  OutPut : 3

import java.util.*;

class Assignment2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of Rows :");
        int iRows = sobj.nextInt();

        System.out.println("Enter the number of Colomus :");
        int iCols = sobj.nextInt();

        int Arr[][] = new int[iRows][iCols];

        System.out.println("Enter the Elements of the matrix :");
        for(int i = 0; i < iRows; i++)
        {
            for(int j = 0; j < iCols; j++)
            {
                Arr[i][j] = sobj.nextInt();
            }
        }

        System.out.println("Elements Matrix are :");
        for(int i = 0; i < iRows; i++)
        {
            for(int j = 0; j < iCols; j++)
            {
                System.out.print(Arr[i][j]+"\t");
            }
            System.out.println("");
        }
    
        System.out.println("Enter the Number that Number Check Frequency of that number :");
        int Freq = sobj.nextInt();

        int iCount = 0;
        for(int i = 0; i < iRows; i++)
        {
            for(int j = 0; j < iCols; j++)
            {
                if(Arr[i][j] == Freq)
                {
                    iCount++;
                }
            }
        }

        System.out.println("Frequency of that number is :" +iCount);

    }
}